lang/python/python-paramiko/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-paramiko
+PKG_VERSION:=2.7.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=paramiko
+PKG_HASH:=920492895db8013f6cc0179293147f830b8c7b21fdfc839b6bad760c27459d9f
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+PKG_LICENSE:=LGPL-2.1-only
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PYTHON3_PKG_SETUP_ARGS:=
+
+define Package/python3-paramiko
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=SSH2 protocol library
+  URL:=https://github.com/paramiko/paramiko/
+  DEPENDS:=+python3-light +python3-bcrypt +python3-pynacl
```

Comment:
[python3-cryptography](https://github.com/paramiko/paramiko/blob/2.7.1/setup.py#L91) is also required.

(FYI when 3.0 is released, bcrypt and pynacl [will become](https://github.com/paramiko/paramiko/blob/2.7.1/sites/www/installing.rst#paramiko-itself) optional dependencies.)

These also appear to be required:
* [python3-logging](https://github.com/paramiko/paramiko/blob/2.7.1/paramiko/common.py#L22)
* python3-openssl (for [hashlib](https://github.com/paramiko/paramiko/blob/2.7.1/paramiko/transport.py#L31))